lib/console.js

```diff
@@ -12,6 +12,10 @@ function Console(stdout, stderr) {
   if (!stderr) {
     stderr = stdout;
   }
+  if (typeof stderr.write !== 'function') {
```

Comment:
To be more explicit about what's going on, mind putting this in an `if else` with the above? Since if `stderr == stdout` there's no need to check `stderr.write`.